mask.json

```diff
@@ -48,6 +49,13 @@
                 ]
             },
             "default": null
+        },
+        "invert": {
+            "description": "If set to `true`, inverts the mask so that the behavior changes as follows:\n\n* **Raster data cube**: Parallel pixels among `data` and `mask` are compared and those pixels in `data` are replaced, which are **zero** (for numbers) or **`false`** (for boolean values) in `mask`.\n* **GeoJSON or vector data cube**: All pixels for which the point at the pixel center does **not** intersect with the corresponding polygon (as defined in the Simple Features standard by the OGC) are replaced.",
```

Comment:
> does **not** intersect with the corresponding polygon

does **not** intersect with any polygon

Reply:
Thanks for spotting this, was copy/pasted from aggregate_polygon obviously. Will be fixed. ;-)